xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -444,6 +444,9 @@ bool CWinSystemWin32::ChangeResolution(RESOLUTION_INFO res)
 {
   const MONITOR_DETAILS &details = GetMonitor(res.iScreen);
 
+  if (!&details)
```

Comment:
this reads strange, as it's a reference it shouldn't be null and it's not a reference to a pointer either.